examples/platforms/nrf528xx/src/radio.c

```diff
@@ -144,6 +147,23 @@ static uint32_t        sAckFrameCounter;
 static uint8_t         sAckKeyId;
 #endif
 
+static int8_t GetTransmitPowerForChannel(uint8_t aChannel)
+{
+    int8_t channelMaxPower = nrf5GetChannelMaxTransmitPower(aChannel);
+    int8_t power           = 0; // 0 dbm as default value
+
+    if (sPowerInitialized)
+    {
+        power = OT_MIN(channelMaxPower, sDefaultTxPower);
```

Comment:
Does `sDefaultTxPower` mean the TX power at the antenna or just the output of the radio chip?

Reply:
All the power is meant to be the power at the antenna. If you have FEM/antenna you should modify the output in the function `nrf_802154_fal_tx_power_get`